packages/cursorless-vscode/src/ide/vscode/VscodeFocusEditor.ts

```diff
@@ -32,6 +32,11 @@ export default async function vscodeFocusEditor(
   ide: VscodeIDE,
   editor: VscodeTextEditorImpl,
 ) {
+  // Focusing the search editor brings focus back to the input field.
+  if (editor.document.uri.scheme === "search-editor") {
+    return;
+  }
+
```

Comment:
is there a way to detect that the search editor is already focused? If it is not, maybe we still want to issue this command? Either way, I'd add a FIXME here because this code makes it look like it's doing the right thing, when really it's a hack. I'd like to the issue

Reply:
Comment added